clusterloader2/run-e2e.sh

```diff
@@ -23,14 +23,14 @@ export KUBECONFIG="${KUBECONFIG:-${HOME}/.kube/config}"
 export KUBEMARK_ROOT_KUBECONFIG="${KUBEMARK_ROOT_KUBECONFIG:-${HOME}/.kube/config}"
 
 # Deploy the GCP PD CSI Driver if required
-if [[ "${DEPLOY_GCI_DRIVER:-false}" == "true" ]]; then
+if [[ "${DEPLOY_GCI_DRIVER:-true}" == "true" ]]; then
```

Comment:
I don't think we can change default here. This script is used also outside of gcp.

I think we should keep default and set this flag in our tests specifically.

Reply:
make sense, can you give me a pointer on where to change this flag? it's nowhere to be found in `perf-tests` repo

Reply:
You can set any env variable for a given test in test-infra repo.

Here I recommend changing this in preset-e2e-scalability-periodics-master preset which will automatically affect all periodic jobs using master branch:

https://github.com/kubernetes/test-infra/blob/2cb4ec605eed0c0db0c624a2990779fff43114b4/config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml#L314

Then you should add this variable to other presets as described in https://github.com/kubernetes/test-infra/pull/19710#issue-730566534

The potential problem is that once we add this to preset-e2e-scalability-periodics (as described there), this will affect also release presubmits in file: 

https://github.com/kubernetes/test-infra/blob/c3b32daef3348869e9c4621b875c8de07120622b/config/jobs/kubernetes/sig-release/release-branch-jobs/1.22.yaml#L370

which are also using this presubmit. Setting this env in that file will try to use perf-tests/release-1.22 version of this code DEPLOY_GCI_DRIVER which isn't intended.

I recommend manually adding --env= DEPLOY_GCI_DRIVER=false in all release-branch-jobs/1.X.yaml that are using preset-e2e-scalability-periodics to override this preset value.

I know it's quite complicated, we can figure out details during test-infra PR reviews.